docs/concepts/policy/pod-security-policy.md

```diff
@@ -207,37 +207,58 @@ podsecuritypolicy "permissive" deleted
 
 ## Enabling Pod Security Policies
 
-In order to use Pod Security Policies in your cluster you must ensure the
-following
+In order to use Pod Security Policies in your cluster:
 
-1.  You have enabled the API type `extensions/v1beta1/podsecuritypolicy` (only for versions prior 1.6)
-1.  [You have enabled the admission control plug-in `PodSecurityPolicy`](/docs/admin/admission-controllers/#how-do-i-turn-on-an-admission-control-plug-in)
-1.  You have defined your policies
+1. Enable the API type `extensions/v1beta1/podsecuritypolicy` (only for versions prior 1.6)
+1. [Enable the admission control plug-in `PodSecurityPolicy`](/docs/admin/admission-controllers/#how-do-i-turn-on-an-admission-control-plug-in)
+1. Create PodSecurityPolicy objects
+1. Authorize users, groups, or service accounts to perform the `use` verb on desired policies
 
 ## Working With RBAC
 
 In Kubernetes 1.5 and newer, you can use PodSecurityPolicy to control access to
 privileged containers based on user role and groups. Access to different
 PodSecurityPolicy objects can be controlled via authorization.
 
-Note that [Controller Manager](/docs/admin/kube-controller-manager/) must be run
-against [the secured API port](/docs/admin/accessing-the-api/), and must not
-have superuser permissions. Otherwise requests would bypass authentication and
-authorization modules, all PodSecurityPolicy objects would be allowed,
-and user will be able to create privileged containers.
+Before beginning to grant access to specific policies, ensure users and service accounts have not been given global access
+by running the following commands as a privileged user.
+
+Check anonymous users (should answer `no`):
+```shell
+kubectl auth can-i use podsecuritypolicies --as=system:anonymous --as-group=system:unauthenticated
+```
+
+Check all authenticated users (should answer `no`):
+```shell
+kubectl auth can-i use podsecuritypolicies --as=any-random-user --as-group=system:authenticated
+```
+
+Check all service accounts (should answer `no`):
+```shell
+kubectl auth can-i use podsecuritypolicies --as=system:serviceaccount:any-random-ns:any-random-sa
+```
+
+If any of those checks answer `yes`, it means that the associated set of users has been authorized
+to use any defined PodSecurityPolicy. Locate and remove that permission before continuing.
 
 PodSecurityPolicy authorization uses the union of all policies available to the
-user creating the pod and
-[the service account specified on the pod](/docs/tasks/configure-pod-container/configure-service-account/).
+user creating the pod and [the service account specified on the pod](/docs/tasks/configure-pod-container/configure-service-account/).
+This means that if the control loops started by the [Controller Manager](/docs/admin/kube-controller-manager/)
```

Comment:
> control loops started by 

This reads a little bit too much about the implementation details. Users don't have control how each control loop is started, IIRC. So ... I'd vote for the original description -- don't start kcm using a superuser account or something like that.